diff --git a/api.js b/api.js
index f4ecb91959f62fc938416a0585458c0a710af161..b6aca8de6d2a8bbb8a84965fdb06da9eb6ebbdde 100644
--- a/api.js
+++ b/api.js
@@ -25345,9 +25345,9 @@ AND
         progressCallback("checks", checksCount, "fetching");
       }
       for (const checkConstraintRow of checkConstraints) {
-        const constraintName = checkConstraintRow["CONSTRAINT_NAME"];
-        const constraintValue = checkConstraintRow["CHECK_CLAUSE"];
-        const tableName = checkConstraintRow["TABLE_NAME"];
+        const constraintName = checkConstraintRow["CONSTRAINT_NAME"] ?? checkConstraintRow["constraint_name"];
+        const constraintValue = checkConstraintRow["CHECK_CLAUSE"] ?? checkConstraintRow["check_clause"];
+        const tableName = checkConstraintRow["TABLE_NAME"] ?? checkConstraintRow["table_name"];
         const tableInResult = result[tableName];
         tableInResult.checkConstraint[constraintName] = {
           name: constraintName,
diff --git a/api.mjs b/api.mjs
index a940272dfb05accf57a472c46c52652ec9ad0080..cc6a98df0a6c96cc944989664496e1f3a1adee9d 100644
--- a/api.mjs
+++ b/api.mjs
@@ -25374,9 +25374,9 @@ AND
         progressCallback("checks", checksCount, "fetching");
       }
       for (const checkConstraintRow of checkConstraints) {
-        const constraintName = checkConstraintRow["CONSTRAINT_NAME"];
-        const constraintValue = checkConstraintRow["CHECK_CLAUSE"];
-        const tableName = checkConstraintRow["TABLE_NAME"];
+        const constraintName = checkConstraintRow["CONSTRAINT_NAME"] ?? checkConstraintRow["constraint_name"];
+        const constraintValue = checkConstraintRow["CHECK_CLAUSE"] ?? checkConstraintRow["check_clause"];
+        const tableName = checkConstraintRow["TABLE_NAME"] ?? checkConstraintRow["table_name"];
         const tableInResult = result[tableName];
         tableInResult.checkConstraint[constraintName] = {
           name: constraintName,
diff --git a/bin.cjs b/bin.cjs
index 07274ea6f53fd54a129b3d338892feda6e995cff..167634083758b4500215205867e28cb5c2f2f6d1 100755
--- a/bin.cjs
+++ b/bin.cjs
@@ -16683,9 +16683,9 @@ AND
         progressCallback("checks", checksCount, "fetching");
       }
       for (const checkConstraintRow of checkConstraints) {
-        const constraintName = checkConstraintRow["CONSTRAINT_NAME"];
-        const constraintValue = checkConstraintRow["CHECK_CLAUSE"];
-        const tableName = checkConstraintRow["TABLE_NAME"];
+        const constraintName = checkConstraintRow["CONSTRAINT_NAME"] ?? checkConstraintRow["constraint_name"];
+        const constraintValue = checkConstraintRow["CHECK_CLAUSE"] ?? checkConstraintRow["check_clause"];
+        const tableName = checkConstraintRow["TABLE_NAME"] ?? checkConstraintRow["table_name"];
         const tableInResult = result[tableName];
         tableInResult.checkConstraint[constraintName] = {
           name: constraintName,
